airbyte-webapp/build.gradle

```diff
@@ -71,6 +71,10 @@ task buildStorybook(type: NpmTask) {
     inputs.dir 'src'
 
     outputs.dir 'build/storybook'
+
+    environment = [
+        'NODE_OPTIONS': '--max_old_space_size=4096'
```

Comment:
:information_source: It seems storybooks compile process with Vite requires more memory. I saw CI failing with an heap out of memory error on this, so giving it more space here.